test/blackbox-tests/test-cases/install/install-no-prefix.t

```diff
@@ -0,0 +1,22 @@
+Here we specify the behaviour of dune install when no install prefix is given.
+
+  $ cat > dune-project << EOF
+  > (lang dune 3.9)
+  > EOF
+
+  $ cat > dune << EOF
+  > (executable
+  >  (public_name main))
+  > EOF
+
+  $ cat > main.ml
+  $ cat > proj.opam
+
+  $ dune build @install
+
+We should suggest to the user here that they should use --prefix.
+
+  $ dune install
```

Comment:
This only works some of the time when dune doesn't detect an install location. I am not sure how to make this reproducible. Perhaps it is better to just remove this test. WDYT @emillon?

Reply:
I believe that unsetting `OPAM_SWITCH_PREFIX` will make `dune install` require an explicit `--prefix`.